rasa/nlu/emulators/luis.py

```diff
@@ -1,50 +1,77 @@
 from typing import Any, Dict, Text
 
 from rasa.nlu.emulators.emulator import Emulator
+from rasa.shared.nlu.constants import (
+    ENTITIES,
+    ENTITY_ATTRIBUTE_TYPE,
+    ENTITY_ATTRIBUTE_ROLE,
+    ENTITY_ATTRIBUTE_VALUE,
+    ENTITY_ATTRIBUTE_END,
+    ENTITY_ATTRIBUTE_START,
+    INTENT_RANKING_KEY,
+    TEXT,
+    INTENT,
+    INTENT_NAME_KEY,
+    PREDICTED_CONFIDENCE_KEY,
+)
 from typing import List, Optional
 
 
 class LUISEmulator(Emulator):
-    """Emulates Luis responses."""
+    """Emulates the response format of the LUIS Endpoint API v3.0 /predict endpoint.
+
+    https://westcentralus.dev.cognitive.microsoft.com/docs/services/luis-endpoint-api-v3-0/
+    """
 
     def _top_intent(self, data) -> Optional[Dict[Text, Any]]:
-        if data.get("intent"):
-            return {
-                "intent": data["intent"]["name"],
-                "score": data["intent"]["confidence"],
-            }
-        else:
+        intent = data.get(INTENT)
+
+        if not intent:
             return None
 
-    def _ranking(self, data) -> List[Dict[Text, Any]]:
-        if data.get("intent_ranking"):
+        return {
+            "intent": intent[INTENT_NAME_KEY],
+            "score": intent[PREDICTED_CONFIDENCE_KEY],
+        }
+
+    def _intents(self, data) -> List[Dict[Text, Any]]:
+        if data.get(INTENT_RANKING_KEY):
             return [
```

Comment:
according to the linked documentation page this should return a `Dict` and not a `List`